### ISSUE TITLE
Path definition

### ISSUE THREAD

**Issue by Dogway** [animetosho/par2cmdline-turbo#8]
Sorry to interrupt but I'm unable to make this work in almost all conditions in Windows.

Don't Work:
Error log: "You must specify a list of files when creating."
`"D:\exepath\par2.exe" c -s5120000 -r5 -v "Q:\AnotherPath\PAR2\filename.par2" "Q:\FilePath\filename.mp4"`
`"D:\exepath\par2.exe" c -s5120000 -r5 -v -B "Q:\FilePath" "Q:\AnotherPath\PAR2\filename.par2" "filename.mp4"`

Do Work:
`"D:\exepath\par2.exe" c -s5120000 -r5 -v "filename.par2" "filename.mp4"`
`"D:\exepath\par2.exe" c -s5120000 -r5 -v "Q:\FilePath\filename.mp4"`

In other words, I'm unable to choose where to save the par2 files.

**Comment by animetosho** [comment]
Could you please check if the issue persists with the [upstream par2cmdline](https://github.com/Parchive/par2cmdline) project, and if so, submit the issue there?

**Comment by Dogway** [comment]
Just opened a ticket here https://github.com/Parchive/par2cmdline/issues/180
Anyway the program seems not maintained anymore, last build is over 3 years now.

**Comment by animetosho** [comment]
Thanks!  
Yeah, the current par2cmdline maintainers are looking for a new maintainer to carry on the role.

As a workaround to your problem, you can try changing the working directory to where you want the output PAR2 to be.  
Alternatively, you can try [ParPar](https://github.com/animetosho/ParPar) (for create) or [MultiPar](https://github.com/Yutaka-Sawada/MultiPar) (for Windows) clients, which are currently maintained.

**Comment by Dogway** [comment]
> As a workaround to your problem, you can try changing the working directory to where you want the output PAR2 to be. 

How? As stated by default it saves on filename read path. And defining any output path triggers the error listed above.

> Alternatively, you can try [ParPar](https://github.com/animetosho/ParPar) (for create) or [MultiPar](https://github.com/Yutaka-Sawada/MultiPar) (for Windows) clients, which are currently maintained.

They don't work on CUDA. ParPar doesn't repair/verify and MultiPar is a GUI, I need a command line tool to run with BATCH/AHK/Python scripts for automation.

**Comment by animetosho** [comment]
> How?

Something like (for your first example):

	cd /D "Q:\AnotherPath\PAR2"
	"D:\exepath\par2.exe" c -s5120000 -r5 -v "filename.par2" "Q:\FilePath\filename.mp4"

And yeah, it looks like a bug, hence this being a workaround.

> They don't work on CUDA

par2cmdline doesn't support CUDA either?  
MultiPar has OpenCL support though.

> MultiPar is a GUI, I need a command line tool 

MultiPar comes with the command-line par2j application.

**Comment by Dogway** [comment]
```
cd /D "Q:\AnotherPath\PAR2"
"D:\exepath\par2.exe" c -s5120000 -r5 -v "filename.par2" "Q:\FilePath\filename.mp4"
```

As I said above this call doesn't work as you are defining the output argument:
> Ignoring out of basepath source file: Q:\FilePath\filename.mp4
> You must specify a list of files when creating.

> par2cmdline doesn't support CUDA either, so I don't get your point.
> MultiPar has OpenCL support though.
Ah yes, sorry, I mixed it with whisper which I'm also using it heavily back to back with par2
